_includes/desktop-builders-setting.md

```diff
@@ -0,0 +1,60 @@
+If you have turned on the Docker Desktop
+[**Builds** view (beta)](/desktop/use-desktop/builds/), you can use the
+**Builders** tab to inspect and manage builders in the Docker Desktop settings.
+
+### Inspect
+
+To inspect builders, find the builder that you want to inspect and select the
+expand icon. You can only inspect active builders.
+
+The detailed builder view shows:
+
+- Builder status
+- Information about the BuildKit runtime
+- Capabilities and driver type
+- Supported platforms
+- Disk usage
+
+### Select a different builder
+
+The **Selected builder** section displays the selected builder.
+To select a different builder:
+
+1. Find the builder that you want to use under **Available builders**
+2. Open the drop-down menu next to the builder's name.
+3. Select **Use** to switch to this builder.
+
+Your build commands now use the selected builder by default.
+
+### Create a builder
+
+To create a builder, use the Docker CLI. See
+[Create a new builder](/build/builders/manage/#create-a-new-builder)
+
+### Remove a builder
+
+You can remove a builder if:
+
+- The builder isn't your [selected builder](/build/builders/#selected-builder)
+- The builder isn't a [default builder](/build/builders/#default-builder),
+  associated with a Docker context.
```

Comment:
I think it would be better to just say "The builder isn't one associated with a Docker context. In this case you need to remove the context using the `docker context rm` command."